backend/routers/vendor.js

```diff
@@ -75,4 +76,17 @@ router
   .patch(VendorController.updateMyProductRequestController, RequestHelper.returnResponse)
   .delete(VendorController.deleteMyProductRequestController, RequestHelper.returnResponse);
 
+router.get("/", VendorController.getAllVendorsController, RequestHelper.returnResponse);
+
+router
```

Comment:
Why does the endpoints related to vendor seem to be added here. Is it related to the squash and merge problem?

Reply:
It seems so, I will remove the duplicate